### DISCUSSION TITLE
API Endpoint for retrieving usage report

### DISCUSSION THREAD

**Discussion by aidandj** [community/community#60251] [Apps, API and Webhooks]
As far as I can tell from documentation it appears that the only way to get a usage report is to press the button in the UI and have it sent to your email. An API endpoint to download the data, or even an API to send that email would help with automating reports and usage metrics.

**Comment by DSDmark** [comment]
Hi @artificial-aidan,

I'm not sure. But, did you means by that: https://docs.github.com/en/rest

**Reply by aidandj** [reply]
I specifically mean an endpoint for retrieving the usage report. That is just the documentation for the rest api
